css-values-4/Overview.bs

```diff
@@ -2588,7 +2588,7 @@ Trigonometric Functions: ''sin()'', ''cos()'', ''tan()'', ''asin()'', ''acos()''
 	all represent the same value,
 	approximately ''.707''.)
 	They all represent a <<number>>;
-	''sin()'' and ''cos()'' will always return a number between 0 and 1,
+	''sin()'' and ''cos()'' will always return a number between -1 and 1,
```

Comment:
Should this use minus sign instead of hyphen, like −∞ in the following line?

```suggestion
	''sin()'' and ''cos()'' will always return a number between −1 and 1,
```